docs/README.md

```diff
@@ -18,18 +18,34 @@ This app integrates the [Adyen for Platforms](https://docs.adyen.com/platforms)
 
 ⚠️ Before you can use the Adyen Platforms app, the [Adyen Payment connector](https://github.com/vtex-apps/connector-adyen) must be installed and configured in you VTEX account.
 
+### Contacting Adyen
+
+Users must first contact Adyen to get their Adyen for Platforms set up. You will know when Adyen for Platforms is setup when a merchant account has the Platform tab on the sidebar. [Adyen for Platforms Quick Start Guide](https://docs.adyen.com/platforms/quick-start)
+
 ### Installing the App
 
 1. Install this app in the desired account using the CLI command `vtex install vtex.adyen-platforms`.
 2. In your admin sidebar, select `Adyen for Platforms` under the **Marketplace** section.
 3. Select the `Settings` tab:
-   - Enter your Platform API credentials. These API credentials are found in your Adyen admin panel, under the **Developers** menu. You will need the credentials for your Marketplace account: ws\_[123456]@MarketPlace.[AccountName])
+   - Enter your Platform API credentials.
+      - Switch from `Company` account to `Merchant` account
+      - Under the **Developers** menu, select `API credentials`
+      - Select the username that follows this format `ws\_[123456]@MarketPlace.[YourPlatformAccount]`, if there is no username in that format, see **Creating New Web Service User**
+      - Use `Generate New API Key` to create a new key for Adyen and store it somewhere safe. This can be regenerated if lost
    - Enter your production Adyen for Platforms API endpoint. For testing, you can enter the test endpoint: `https://cal-test.adyen.com/cal/services`. This is also found under the **Developers** menu in your Adyen account.
    - Enter the URL sub-merchants will be directed to when they complete their onboarding. This is optional, see the Onboarding section for details on this process.
 
 ### Managing Seller Accounts
 
-The Sellers tab in the Adyen for Platforms menu in your VTEX admin will display all Seller accounts in your VTEX marketplace. Selecting a seller will bring you to that seller's detail screen.
+The Sellers tab in the Adyen for Platforms menu in your VTEX admin will display all Seller accounts in your VTEX marketplace.
+- Users can `Create Adyen Account` for specific sellers
+  1. Enter the Adyen Account Holder Code
+      - Ensure you're on Adyen's `Merchant` account in Adyen's website
+      - Under the `Platform` tab, select `Sub-merchants`. Select the desired `Adyen Account Holder Code`
```

Comment:
Is this accurate? I thought users could enter whatever account holder code they wanted, and then AFTER submitting, it would show up in the Sub-Merchants list inside Adyen

Reply:
That's right. I'll correct it.